docs/tutorials/Aragon_Core_User_Guide.md

```diff
@@ -0,0 +1,197 @@
+
+“Vote Duration” is the length of time that the vote will be open for participation. For example, if the Vote Duration is set to 24 Hours (H), then tokenholders have 24H to participate in the vote. After the 24 hours is over, if the vote has not been approved by a majority of tokenholders and/or does not make quorum, then the vote will automatically fail. If the vote makes quorum and receives approval by a majority of tokenholders by the end of the Vote Duration, then the vote will automatically pass.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/democracysettings.png)
+
+**Step 5. Choose token name and symbol then click “Next”**
+
+TODO: note character limits
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daotokens.png)
+
+**Step 6. Sign organization creation transactions**
+
+Now you need to sign two transactions in MetaMask to create your organization. Open MetaMask if the wallet window does not open automatically. Set the gas limit for each transaction at 6,660,000 to ensure that the transactions succeed. The second transaction may show up in MetaMask as a transaction with a blue dot next to it, indicating that it needs to be signed. Click the blue dot to open the transaction and sign it.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daomm1.png)
+
+**Step 7. Click “Get Started”**
+
+Your new Democracy DAO is ready to go!
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/orgready.png)
+
+### 2. Explore the Democracy DAO
+
+#### Home app
+
+The Home app welcomes you to your DAO and offers you several options for different actions you can take to interact with your DAO. You can also use the menu on the left-hand side of the screen to view the different apps that are currently installed for your DAO. Aragon Core currently has the Home, Token Manager, Voting, Finance, and Settings apps installed, with a Permissions and Apps Manager coming soon.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daohome.png)
+
+#### Token Manager app
+
+The Token Manager is the app you use to mint new tokens and assign them to yourself or other Ethereum accounts.
+
+![](https://raw.githubusercontent.com/aragon/aragon-wiki/master/docs/images/user_guide/daotokenmanager.png)
+
+You can mint new tokens by following these steps:
+
+**Step 1. Click “Assign Tokens” button**
+
```

Comment:
Maybe add a note/warning about giving up the control of the DAO is a possibility here

Reply:
I prefer to iterate with small changes like this after this pull request is merged.